### DISCUSSION TITLE
How to filter single button events

### DISCUSSION THREAD

**Discussion by ArcadeD3vil** [bxparks/AceButton#70] [General]
Hi there,
i´m just trying your code and i love that.
for my project i´m trying to use the "one button with pulldown" script.
in general the output can differ between single, double click, long press and long repeat press.
my goal is depending on eventType to decide what to do.
but the value for "DoubleClick" is sending 1 or 2 next to 3 everytime.
so i can´t use 1 or 2 next the double click.
is it possible only to get the value "3" for the "DoubleClick" feature when double click the button?

**Comment by bxparks** [comment]
Hi, I don't think I understand your question... You first mention "double long press" which is confusing. AceButton handles "double click" and "long press", but not "double long press". Then you ask about DoubleClick, which is not how you get RepeatPress. 

Can you see if one of the following sections in the README helps you?

* https://github.com/bxparks/AceButton#ClickedAndDoubleClicked
* https://github.com/bxparks/AceButton#PressedAndLongPressed

After that, can you ask your question again, and be more precise about which events you want to detect, and what you want to happen when they are detected?
